test/plugins/iamauth.js

```diff
@@ -479,39 +466,6 @@ describe('#db IAMAuth Plugin', function() {
     });
   });
 
-  it('skips authentication renewal on 401 response if previous attempts failed', function(done) {
```

Comment:
Correct me if I'm wrong - this test is no longer valid because on a `401` response from a Cloudant request we will attempt to renew the auth, but if that renewal fails the error is now returned to the user so we need no longer check that auth renewal is subsequently skipped.